.github/workflows/lint.yaml

```diff
@@ -30,3 +30,4 @@ jobs:
         shell: Rscript {0}
         env:
           LINTR_ERROR_ON_LINT: true
+          R_LINTR_LINTER_FILE: .ci/.lintr
```

Comment:
missing file extension, no?

Reply:
No -- {lintr} looks for `${R_LINTR_LINTER_FILE}.R` then `${R_LINTR_LINTER_FILE}`:

https://github.com/r-lib/lintr/blob/bad1632f23b77f41cec13fb6f1c883302f64f506/R/settings_utils.R#L90-L91